### DISCUSSION TITLE
audio output to pipe; No "command" parameter specified

### DISCUSSION THREAD

**Discussion by Staphylea** [MusicPlayerDaemon/MPD#1745] [General]
Hey, I try to setup a pipe output. 
What I´ve done is
`audio_output {
       type            "pipe"
       name            "/tmp/snapfifo"
       mixer_type      "software"
       format          "48000:16:2"
}
`
When I try to start `systemctl start mpd.service` it fails with
` exception: Failed to configure output in line 367; No "command" parameter specified`

I´ve checked [Configuring audio outputs](https://mpd.readthedocs.io/en/latest/user.html#configuring-audio-outputs)
Can´t find any description about 'command' param.

Found some text under [pipe](https://mpd.readthedocs.io/en/latest/plugins.html#pipe)

checked the man mpd.conf from [required](https://mpd.readthedocs.io/en/latest/mpd.conf.5.html#required-audio-output-parameters)
coudn´t find anything under requried or optional parameters.

In the plugin reference from [fifo](https://mpd.readthedocs.io/en/latest/plugins.html#fifo)
only the 'path' parameter is described.

The 'command' parameter seems to be mandatory for pipe output. But I have no idea which command should be executed at this point. What would be your recommendation? 

Thanks

**Comment by skidoo23** [comment]
Snapcast requires fifo output not pipe.

**Reply by MaxKellermann** [reply]
Also don't use the fifo output - MPD has a native Snapcast output plugin.

**Reply by Staphylea** [reply]
omg, thank you both to point out the obvious. 
think my brain mixed up 'a named pipe' with pipe 
@MaxKellermann I didn´t noticed that yet. It´s working like a charm.

**Reply by skidoo23** [reply]
> Also don't use the fifo output - MPD has a native Snapcast output plugin.

But Android Snapcast App - tested with recent version 0.27 - is not able to connect to MPD (from git master) Snapcast server output.
